odbc_fdw.c

```diff
@@ -1343,6 +1336,8 @@ odbcBeginForeignScan(ForeignScanState *node, int eflags)
 static TupleTableSlot *
 odbcIterateForeignScan(ForeignScanState *node)
 {
+    EState	   *estate = node->ss.ps.state;
```

Comment:
Ah, OK, the name comes from the data structure name, EState: http://doxygen.postgresql.org/execnodes_8h_source.html#l00354